controllers/articles.controllers.js

```diff
@@ -0,0 +1,20 @@
+const {selectArticleById} = require("../models/articles.models.js")
+
+function getArticles (req,res,next){
+    const {article_id} = req.params;
+    
+    if (isNaN(article_id)) {
+        return res.status(400).send({ msg: 'Bad request' });
+      }
+
+return selectArticleById(article_id)
+.then(article => {
+    if (!article) {
+        return res.status(404).send({ msg: 'Article not found' });
```

Comment:
This works, but it would be nice to keep the controller clean. So it's better to use Promise.reject in the model, the error will be caught in the catch block here, and will be passed down to the error handler if you've got one in app.js